src/ops/array.rs

```diff
@@ -271,7 +278,7 @@ pub(crate) fn cmp_internal(left: &[u32; 3], right: &[u32; 3]) -> core::cmp::Orde
 }
 
 #[inline]
-pub(crate) fn is_all_zero(bits: &[u32]) -> bool {
+pub(crate) fn is_all_zero<const N: usize>(bits: &[u32; N]) -> bool {
     bits.iter().all(|b| *b == 0)
 }
```

Comment:
I **think** this will help the compiler unroll the loop. Not sure if it is already optimizing but such addition doesn't hurt.

The same reasoning applies to the other `<const N: usize>` functions

Reply:
Since this function is inline, it actually was unrolled (I checked on master).